integration-tests/bats/sql-server.bats

```diff
@@ -1357,7 +1357,7 @@ databases:
     dolt commit -am "create three tables"
 
     cd ..
-    start_sql_server_with_args --user dolt -ltrace --no-auto-commit
+    start_sql_server_with_args --host 0.0.0.0 --user dolt -ltrace --no-auto-commit
```

Comment:
What does the `0.0.0.0` address do for `dolt sql-server`? I looked in the docs and didn't see anything. Does this have a special meaning?

Reply:
the client uses --host 0.0.0.0 to connect to this server in `sql-server-mysql.expect` file, so when I added unix socket support, it started failing, to match the host I added this here, the client host can be modified instead I think.